--- os_unix.c	2020-05-02 00:34:08.000000000 +0800
+++ os_unix.c.patched	2020-05-02 00:35:18.000000000 +0800
@@ -114,10 +114,11 @@
 #if defined(__APPLE__) && ((__MAC_OS_X_VERSION_MIN_REQUIRED > 1050) || \
                            (__IPHONE_OS_VERSION_MIN_REQUIRED > 2000))
 #  if (!defined(TARGET_OS_EMBEDDED) || (TARGET_OS_EMBEDDED==0)) \
-       && (!defined(TARGET_IPHONE_SIMULATOR) || (TARGET_IPHONE_SIMULATOR==0))
+       && (!defined(TARGET_IPHONE_SIMULATOR) || (TARGET_IPHONE_SIMULATOR==0)) \
+       && !defined(TARGET_OS_UIKITFORMAC)
 #    define HAVE_GETHOSTUUID 1
 #  else
-//#    warning "gethostuuid() is disabled."
+#    warning "gethostuuid() is disabled."
 #  endif
 #endif
 
